app/controllers/daily_cards_controller.rb

```diff
@@ -0,0 +1,20 @@
+class DailyCardsController < ApplicationController
+
+  def eligibility_check
+    current_time = Time.now
+    last_card = User.find(params[:user_id]).timestamp_of_last_daily_card
+    if last_card.nil?
+      last_card = Time.now
+    end
+
+    time_elapsed = current_time - last_card
+    if time_elapsed >= 86400
```

Comment:
```suggestion
    seconds_in_a_day = 86400
    if time_elapsed >= seconds_in_a_day
```
helps to easier understand what that number represents